aexpect/client.py

```diff
@@ -415,7 +416,7 @@
             os.write(proc_input_pipe, cont.encode(self.encoding))
             os.close(proc_input_pipe)
         except OSError:
-            pass
+            LOG.warning("Failed to send '%s'", cont)
```

Comment:
Interesting warning. I don't think it's too important right now but we might want to write a scrubber of strings such as passwords so that they are not logged.

Reply:
Given the purpose of this tool it's IMO important to log the plaintext password.